internal/ocm/client.go

```diff
@@ -0,0 +1,175 @@
+/*
+Copyright (c) 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ocm
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/rosa/pkg/config"
```

Comment:
We should not re introduce rosa as a dependency to rhcs though, we need to move things into ocm-common or setup it up internally to rhcs itself if we believe it will eventually diverge in terms of functionality